uxarray/plot/accessor.py

```diff
@@ -27,232 +34,195 @@ def __init__(self, uxgrid: Grid) -> None:
+
+        Raises
+        ------
+        ValueError
+            If the provided `element` is not one of the accepted options.
         """
-        return grid_plot.mesh(
-            self._uxgrid,
-            backend=backend,
-            exclude_antimeridian=exclude_antimeridian,
-            width=width,
-            height=height,
-            **kwargs,
-        )
 
-    @functools.wraps(grid_plot.mesh)
-    def edges(
-        self,
-        backend: Optional[str] = "bokeh",
-        exclude_antimeridian: Optional[bool] = False,
-        width: Optional[int] = 1000,
-        height: Optional[int] = 500,
-        **kwargs,
-    ):
-        """Vector Line Plot of the edges that make up each face. Equivalent to
-        ``Grid.plot.mesh()``
+        uxarray.plot.utils.backend.assign(backend)
 
-        Parameters
-        ----------
-        backend: str
-            Selects whether to use Holoview's "matplotlib" or "bokeh" backend for rendering plots
-         exclude_antimeridian: bool,
-            Whether to exclude edges that cross the antimeridian
-        height: int
-            Plot Height for Bokeh Backend
-        width: int
-            Plot Width for Bokeh Backend
-        """
-        return grid_plot.mesh(
-            self._uxgrid,
-            backend=backend,
-            exclude_antimeridian=exclude_antimeridian,
-            width=width,
-            height=height,
-            **kwargs,
-        )
+        if element in ["nodes", "corner nodes", "node_latlon"]:
+            lon, lat = self._uxgrid.node_lon.values, self._uxgrid.node_lat.values
+        elif element in ["faces", "face centers", "face_latlon"]:
+            lon, lat = self._uxgrid.face_lon.values, self._uxgrid.face_lat.values
+        elif element in ["edges", "edge centers", "edge_latlon"]:
+            lon, lat = self._uxgrid.edge_lon.values, self._uxgrid.edge_lat.values
+        else:
+            raise ValueError("TODO: ")
 
-    @functools.wraps(grid_plot.node_coords)
-    def node_coords(
-        self,
-        backend: Optional[str] = "bokeh",
-        width: Optional[int] = 1000,
-        height: Optional[int] = 500,
-        **kwargs,
-    ):
-        """Vector Point Plot of Nodes (latitude & longitude of the nodes that
-        define the corners of each face)
+        verts = {"lon": lon, "lat": lat}
 
-        Parameters
-        ----------
-        backend: str
-            Selects whether to use Holoview's "matplotlib" or "bokeh" backend for rendering plots
-         exclude_antimeridian: bool,
-            Whether to exclude edges that cross the antimeridian
-        height: int
-            Plot Height for Bokeh Backend
-        width: int
-            Plot Width for Bokeh Backend
-        """
+        points_df = pd.DataFrame.from_dict(verts)
 
-        return grid_plot.node_coords(
-            self._uxgrid, backend=backend, width=width, height=height, **kwargs
-        )
+        return points_df.hvplot.points("lon", "lat", **kwargs)
```

Comment:
I am a bit confused, starting with this function in `accessor.py`, it does not any longer act like accessor, the functions added with this PR seem to have their own implementations rather than behaving as accessors to the actual functions of `dataarray_plot.py` and `grid_plot.py`. 

Could you please clarify?

Reply:
I don't believe `dataarray_plot` and `grid_plot` will be necessary moving forward with these changes. The previous separation was consistent with `xarray`, however now that we are much more elegantly wrapping `hvPlot`, we don't need to have it split up as much as we did before. 

It still acts as an accessor (i.e. `GridPlotAccessor` and `UxDataArrayPlotAccessor` are the actual accessor classes) that are attached to the `.plot` attribute of their respective objects.

Reply:
Thanks for clarifying! However, then it is still confusing to see we still keep functions in `grid_plot.py` and `dataarray_plot.py` that will no longer be used, e.g. grid_plot.mesh() etc. 

I get that in a future PR (likely when we are addressing the deprecation of functions like `rasterize`), we will remove `grid_plot.py` and `dataarray_plot.py` fully, but even for the meantime, we need to get rid of those functions they are now implemented in `accessor.py` and will no longer be called from either of those "_plot.py" scripts.

Does that make sense?

Reply:
This makes complete sense, I forgot to remove the functions that aren't called anymore.
* Nothing is being called from `grid_plot.py`, so I have removed that module.
* `dataarray_plot.py` has been reduced to only include the functions that are planned to be deprecated. 

Thanks for clarifying!

Reply:
Sounds good!